src/main/java/duke/AddDeadlineCommand.java

```diff
@@ -0,0 +1,42 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Handles addition of deadline-based Tasks.
+ */
+public class AddDeadlineCommand extends Command {
+    /** duke.Command details in the form [TYPE, INFORMATION] */
+    private final String[] instructions;
+
+    /**
+     * Constructor for duke.AddDeadlineCommand.
+     * @param instructions Contains description and deadline.
+     */
+    public AddDeadlineCommand(String[] instructions) {
+        super();
```

Comment:
`super()` with no argument will be called automatically. Perhaps can omit the `super()` here.